optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/VehicleRouting/VehicleRouting-section.adoc

```diff
@@ -177,7 +177,7 @@ There are several technologies available to precalculate road costs, such as htt
 
 image::UseCasesAndExamples/VehicleRouting/integrationWithRealMaps.png[align="center"]
 
-There are also several technologies to render it, such as http://leafletjs.com[Leaflet] and https://developers.google.com/maps/[Google Maps for developers]: the `optaplanner-webexamples-*.war` has an example which demonstrates such rendering:
+There are also several technologies to render it, such as http://leafletjs.com[Leaflet] and https://developers.google.com/maps/[Google Maps for developers]. The https://www.optaplanner.org/learn/rhpds/optaweb-vehicle-routing.html[OptaWeb Vehicle Routing project] has an example which demonstrates such rendering:
```

Comment:
We've been asked not to link RHPDS in community docs, which makes sense because only partners and internals can access the RHPDS instance.
Please link https://github.com/kiegroup/optaweb-vehicle-routing instead.

Reply:
In that case, I don't understand why we ever published this on the community website:
https://www.optaplanner.org/learn/rhpds/optaweb-vehicle-routing.html

It's the first non-github link you get when googling "OptaWeb".